client/src/components/Burger/Burger.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import "./Burger.css";
+
+// Class Based React Component
+class Burger extends Component {
```

Comment:
I'm not sure this needs to be a class component - I think you could accept `active` as a prop to determine whether or not it's active or not. The callback passed in should toggle active/inactive as a boolean (true for active, false for inactive)